docs/timezone.md

```diff
@@ -8,13 +8,68 @@
 
-Since `Temporal.Instant` and `Temporal.PlainDateTime` do not contain any time zone information, a `Temporal.TimeZone` object is required to convert between the two.
+To combine a time zone with a date/time value, and to perform DST-safe operations like "add one day", use [`Temporal.ZonedDateTime`](./zoneddatetime.md).
+
+## Time zone identifiers
+
+Time zones in `Temporal` are represented by string identifiers from the IANA Time Zone Database (like `Asia/Tokyo`, `America/Los_Angeles`, or `UTC`) or a fixed UTC offset like `+05:30`.
+Each time zone has a canonical identifier, and may also have one or more aliases.
+
+Anywhere in ECMAScript that a `Temporal.TimeZone` object is expected, a time zone identifier can be used instead.
+Furthermore, using string identifiers instead of time zone objects allows ECMAScript implementations to perform optimizations that are not possible when using time zone objects.
+Therefore, it's recommended to always use string identifiers instead of time zone objects whenever this is convenient. For example:
+
+```javascript
+inBerlin = Temporal.ZonedDateTime.from('2022-01-28T19:53+01:00[Europe/Berlin]');
+inTokyo = inBerlin.withTimeZone('Asia/Tokyo'); // May be faster / use less RAM
+inTokyo = inBerlin.withTimeZone(Temporal.TimeZone.from('Asia/Tokyo')); // OK, but not optimal
+```
+
+### Canonicalization of time zone identifiers
+
+The `id` property of any `Temporal.TimeZone` instance returns the canonical identifier for that time zone, even if the input used to create that instance used an alias and/or a different mix of upper and lower case letters. For example:
+
+```javascript
+// In Firefox 109, India's time zone's canonical identifier is 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('asia/calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('ASIA/CALCUTTA').id; // => 'Asia/Kolkata'
+
+// In Chrome 109, India's time zone's canonical identifier is 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Calcutta'
+
+// "UTC" is the canonical identifier for all UTC time zone aliases.
+Temporal.TimeZone.from('Etc/UTC').id; // => 'UTC'
+Temporal.TimeZone.from('Etc/GMT').id; // => 'UTC'
+Temporal.TimeZone.from('UTC').id; // => 'UTC'
+
+// Offset time zone identifiers are resolved to a canonical format
+Temporal.TimeZone.from('01').id; // => '+01:00'
+Temporal.TimeZone.from('+01').id; // => '+01:00'
+Temporal.TimeZone.from('+01:00').id; // => '+01:00'
+Temporal.TimeZone.from('+0100').id; // => '+01:00'
+Temporal.TimeZone.from('+01:00:00').id; // => '+01:00'
+```
+
+The canonical identifier for a time zone is guaranteed to be the same for the lifetime of any `Temporal`-using program.
+However, the canonical identifier is not guaranteed to be the same across time or between different ECMAScript implementations.
```

Comment:
I'd phrase it as "the canonical identifier may change in subsequent updates to the time zone database." The discrepancy between ECMAScript implementations is not supposed to happen and I hope it will be fixed (see above.)